pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -214,10 +208,10 @@ func (w *Watcher) traversePluginDir(dir string) error {
 			w.wg.Add(1)
 			go func() {
```

Comment:
don't we have to remove this goroutine, so that `traversePluginDir` runs synchronously and completes before we start our processing loop (and before we continue processing other events in the event loop)?

Reply:
Fixed. I thought starting the goroutine would be enough to guarantee that it'd be executed before the processing goroutine, but I was wrong.